frontend/src/metabase/components/AddToDashSelectDashModal.jsx

```diff
@@ -58,10 +60,21 @@ export default class AddToDashSelectDashModal extends Component {
                     title="Add Question to Dashboard"
                     closeFn={this.props.closeFn}
                 >
+                <div className="flex flex-column">
+                    <div
+                        className="text-brand-hover flex-align-right px4 cursor-pointer"
```

Comment:
@wtfleming If you use the `link` class here you can remove `text-brand-hover` and get the link color behavior you're looking for for the action.